packages/components/Button/src/Button.tsx

```diff
@@ -64,19 +66,25 @@ export const Button = compose<IButtonType>({
       },
       content: { children: content, testID: testID },
       icon: createIconProps(icon),
+      chevron: {},
```

Comment:
Could we not have passed the xml in here? Thus that would avoid needing to pass it in at line 87?

Reply:
I had chosen to use the SvgXml type for chevron because our Svg type doesnt support class param. In this case, just the regular Svg type might be sufficient?